internal/args/unmarshal.go

```diff
@@ -253,15 +220,50 @@ func set(dest reflect.Value, argNameWords []string, value string) error {
 			return &MissingStructFieldError{Dest: dest.Interface()}
 		}
 
-		// try to find the correct field in the struct.
+		// We cannot rely on dest.GetFieldByName() as reflect library is doing deep traversing when using anonymous field.
```

Comment:
Do we want to move this code in our own `fieldByName` function?

Reply:
This is not just a get by field as it should create all pointer along the way this is exactly what set is for.